model/src/util.py

```diff
@@ -1,22 +1,34 @@
+from collections import deque
 import math
 from typing import Any, Optional
 import snntorch as snn
 import torch
 
+MAX_RETAINED_SPIKES = 2
 
-class MovingAverageLIF(snn.LIF):
-    def __init__(self, *args: Any, tau_mean: float, tau_var: float, **kwargs: Any) -> None:
+# Zenke's paper uses a tau_mean of 600s
+TAU_MEAN = 600
```

Comment:
Calling this out again, but scaling the units in a certain way will cause different interactions with the learning rule. If we hope to pick the same values as Zenke and hope it mostly works, this may need to be tuned.